src/index.js

```diff
@@ -1,6 +1,13 @@
 const fibonacci = (n) => {
+  n++;
+  let val = [0, 1];
```

Comment:
Better use `const`, you don't reassign this variable. Use names of variables more descriptive of the function that they will perform.